tests/snippets/stdlib_os.py

```diff
@@ -251,6 +251,69 @@ def __exit__(self, exc_type, exc_val, exc_tb):
 	assert os.path.basename(fname) == FILE_NAME
 	assert os.path.dirname(fname) == tmpdir
 
+	# os.get_blocking, os.set_blocking
+	# TODO: windows support should be added for below functions
+	# os.pipe, 
+	# os.set_inheritable, os.get_inheritable, 
+	if os.name != "nt":
+		rfd, wfd = os.pipe()
```

Comment:
I don't think these tests have to be in this `with` block, as they don't use `tmpdir` for anything